includes/commands/class-mu-migration-base.php

```diff
@@ -33,6 +33,15 @@
 	 * @param array $assoc_args
 	 */
 	protected function process_args( $default_args = array(), $args = array(), $default_assoc_args = array(), $assoc_args = array() ) {
+
+		/**
+		 * Filters the method-defined args.
+		 *
+		 * This filter can be used to determine the folder where files will be
+		 * written to and read from.
+		 */
+		$args = (array) apply_filters( 'mu-migration/process_args/args', $args );
```

Comment:
This method is going to be called from the import command as well, so overriding the args via a filter will also change args from the import command. I think that it could causes issues. Maybe you should pass in an additional parameter with the context, like the command and method name